### DISCUSSION TITLE
最近绳网的委托好少阿，卑微绳匠快吃不拉面了(T^T)

### DISCUSSION THREAD

**Discussion by tayiie** [share121/inter-knot#3279] [其他]
![2320001758](https://github.com/user-attachments/assets/82b14187-40f9-461a-9584-f0210585e463)
![2320001759](https://github.com/user-attachments/assets/26b31eaf-bf71-4397-8fab-2d130a3e1d0d)
###最近提瓦特那边好萧条阿，明明都到海岛了，结果就只有几位旧识委托人下过那边的探索。一个新认识的委托人都没有。
#####虽说在下社恐，可这样拉面都快吃不起了。
####而且好想玩黑神话阿，可是时间该往哪挤呢。
纠结<img src="https://bbs-static.miyoushe.com/static/2023/04/28/b71f9bae78003f98a47f0d6e0765324d_7897172668253324041.png" alt="米仔_休息" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/b71f9bae78003f98a47f0d6e0765324d_7897172668253324041.png" alt="米仔_休息" width="75" >

**Comment by RaphaelHider** [comment]
翁法罗斯的委托你可以关注一下，属于高风险高回报的

**Reply by tayiie** [reply]
那边的委托太可怕了。
'''
崩铁的委托最容易翻车了，逃课是没法逃的。交战是不能撤的，一撤就会到锚点了，回合制的时间是拖沓的。每次接到这种委托，我都是战战兢兢履薄冰，献上了数双熊猫眼的说
'''
